src/pages/iou/steps/MoneyRequestAmountPage.js

```diff
@@ -375,6 +372,14 @@ class MoneyRequestAmountPage extends React.Component {
             .value();
     }
 
+    navigateBack() {
+        if (this.isEditing) {
+            Navigation.goBack(ROUTES.getMoneyRequestConfirmationRoute(this.iouType, this.reportID));
+            return;
+        }
+        Navigation.goBack();
```

Comment:
Unfortunately, no. We will be navigated to the fallback route when the current route is the only one in stack. So, if we are on the first amount page and go back, it will always navigate to confirmation route. If we want to make it one line, we can rewrite it to:
```
Navigation.goBack(this.isEditing ? ROUTES.getMoneyRequestConfirmationRoute(this.iouType, this.reportID) : null);
```

Reply:
Updated to above code

Reply:
I don't know what I was thinking :sweat_smile: Totally makes sense please resolve this